packages/old-templates/graphql-codegen-apollo-angular-template/package.json

```diff
@@ -2,6 +2,8 @@
   "name": "graphql-codegen-apollo-angular-template",
```

Comment:
Please remove the entire directory (old-templates)

Reply:
Done!